tests/spec/fs.read.spec.js

```diff
@@ -87,4 +87,18 @@ describe('fs.read', function() {
       });
     });
   });
+
+  it('should fail to read a file that does not exist', function(done) {
+    var fs = util.fs();
+
+    var fd = 0;
+    var rbuffer = new Filer.Buffer(8);
+    rbuffer.fill(0);
+
+    fs.read(fd, rbuffer, 0, rbuffer.length, 0, function(error, result) {
+      expect(error).to.exist;
```

Comment:
We should also deal with the specific error code. In Filer that would (currently) be `'EBADF'`.  In note that in node, it would be: `Error: ENXIO: no such device or address, read errno: -6, code: 'ENXIO'`.  This would require a fix to our implementation if we want to match 100% (not sure it's that important).